contracts/market/collateral/CollateralEscrow_V1.sol

```diff
@@ -52,6 +62,12 @@ contract CollateralEscrow_V1 is OwnableUpgradeable, ICollateralEscrow {
         _balances[loanID] += amount;
     }
 
+    /**
+     * @notice it withdraws an amount of tokens in a respective loanID if the caller is the owner
```

Comment:
actually only the `TellerDiamond` is able to call this on behalf of the borrower